.github/workflows/pre-commit.yml

```diff
@@ -39,8 +35,9 @@ jobs:
         continue-on-error: true
 
       - name: suggester / pre-commit
+        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
AH I just realized this breaks linting for comments.

Pre-commit will fail but it has continue on failure set to true for review dog to run, and in commits review dog doesn't run so the check will show up as passed.

I don't think there's anyway to fix this within this job, I have to make a separate job that triggers on commit.

Damn it!